features/support/env.rb

```diff
@@ -15,6 +15,8 @@
 require "cucumber/api_steps"
 require "json_spec/cucumber"
 
+require_relative "download_helpers"
```

Comment:
Doesn't cucumber load all files from `features/support` automatically? Or does that only happen before it loads the step definitions but after loading `env.rb`?